drivers/hmis/app/views/hmis_admin/user_groups/edit.haml

```diff
@@ -0,0 +1,17 @@
+- title = "Edit Group #{@user_group.name}"
+- content_for :title, title
+
+= content_for :crumbs do
+  = link_to hmis_admin_user_groups_path do
+    &laquo; Manage User Groups
+
+.o-page
+  .o-page__title
+    %h1= content_for :title
+
+= simple_form_for @user_group, as: :user_group, url: hmis_admin_user_group_path do |f|
+  = f.error_notification
+  .form-inputs.mt-4
+    = render 'form', f: f
+  .form_actions.mt-4
+    = f.button :submit, value: 'Update User Group'
```

Comment:
I know this is getting a whole re-design later, but I'm finding this looking really confusing because the "Submit" button is oddly below the user table, even though its just for updating the name.

 How about this in the interim?
I'd also love for the "users to add" box not being left-aligned, but that template is used in the warehouse too.

```suggestion
-# Form for changing group name
= simple_form_for @user_group, as: :user_group, url: hmis_admin_user_group_path do |f|
  = f.error_notification
  .d-flex.align-items-center.mb-10
    .form-inputs.w-50
      = f.input :name
    .form_actions.ml-4
      = f.button :submit, value: 'Update Group Name'

-# Add/Remove members
= render 'users/user_members_table', item: @user_group, path_to_add_users: hmis_admin_user_group_users_path(@user_group), path_to_delete_item: hmis_admin_user_group_path(@user_group), delete_user_lambda: ->(user) { hmis_admin_user_group_user_path(@user_group, user) }
```

Reply:
<img width="1256" alt="Screen Shot 2023-08-11 at 11 05 43 AM" src="https://github.com/greenriver/hmis-warehouse/assets/5333982/77e690b1-364c-4942-97b3-279ebd80c8ee">

=>

<img width="1269" alt="Screen Shot 2023-08-11 at 11 04 48 AM" src="https://github.com/greenriver/hmis-warehouse/assets/5333982/1517fa34-80b6-40af-9da5-dc4db3f37b1e">

Reply:
Thanks, this was definitely wrong.  Went with your approach.